lib/reporters.js

```diff
@@ -14,6 +15,17 @@ var defaultReporter = function (result) {
   });
   return out.join('\n');
 },
+// Default Partial Grunt reporter
+defaultPartialReporter = function (result) {
```

Comment:
The partial reporter needs to work differently from the others in the fact that it needs to put a full template into the validator, so all line numbers are off when reporting back to the original file, and it doesn't actually validate the original file. It validates a wrapped partial.

The lineOffset from the partial is needed so it will report the correct line in the partial and not the wrapped file.
The files list is needed because we need to report to them the original template file they put in, and not the wrapped partial that was file written to a tmp directory and is the actually that got evaluated.

I've fixed this locally so that it passes in the options to the method as well as the result, allowing for custom reporters to still work with it.